test/expected/empty.js

```diff
@@ -0,0 +1 @@
+//# sourceMappingURL=empty.map
```

Comment:
They get used by the `expectFile` helper. The syntax is confusing, but `expectFile('empty.js').in(result)` actually compares the output with the contents of the corresponding file in `test/expected`